code/vector_3d/vector_3d.py

```diff
@@ -0,0 +1,56 @@
+from math import sqrt
+import warnings
+
+
+class Vector3d:
+
+    def __init__(self, x: float = 1.0, y: float = 1.0, z: float = 1.0):
+        self.x = x
+        self.y = y
+        self.z = z
+
+    def is_equal(self, x: float, y: float, z: float) -> bool:
+        return self.x == x and self.y == y and self.z == z
```

Comment:
Use small value to compare floats:
```python
epsilon = 0.000001
return abs(a - b) <= epsilon
```
Python 3.5 adds the `math.isclose()` and `cmath.isclose()` functions which tell whether two values are approximately equal or “close” to each other.